ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.scss

```diff
@@ -31,24 +34,27 @@
     border-radius: 3px;
   }
 
-  &__content {
+  &__checkbox-wrapper {
+    width: 100%;
+    display: flex;
     flex-direction: row;
-    padding: 0 24px 24px 24px;
+    align-items: center;
   }
 
   &__checkbox {
     margin-right: 8px;
+    padding-top: 1px; // better alignment with rest of content
   }
 
   &__checkbox-label {
-    font-size: 14px;
+    display: flex;
+    font-size: 12px;
     margin-top: auto;
     margin-bottom: auto;
     color: $Grey-500;
-    display: flex;
   }
 
   &__checkbox-label-tooltip {
-    margin-left: 8px;
+    margin-left: 8px !important;
```

Comment:
The popover `:only-child` selector takes precedence otherwise.

Reply:
... why does that `:only-child` selector exist in the first place 🤔

Reply:
Ah, I think the problem here is that the `popover-footer :only-child` and `unconnected-account-alert__footer :only-child` selectors were only intended for the _immediate_ child. i.e. they should have been `popover-footer > :only-child` and `unconnected-account-alert__footer > :only-child` respectively.